Sources/Sandbox.Game/Game/Entities/Blocks/MyBeacon.cs

```diff
@@ -53,6 +53,11 @@ static MyBeacon()
             show.Setter = (x, v) => x.RequestShowInTerminal(v);
             MyTerminalControlFactory.AddControl(show);
 
+            show = new MyTerminalControlOnOffSwitch<MyBeacon>("ShowInToolbarConfig", MySpaceTexts.Terminal_ShowInToolbarConfig, MySpaceTexts.Terminal_ShowInToolbarConfigToolTip);
```

Comment:
Why add this here (and in MyRadioAntenna)? The control should already be added by MyTerminalBlock.